build.sbt

```diff
@@ -1,24 +1,9 @@
 import sbt.Keys.organization
 
 val commonSettings: Seq[SettingsDefinition] = Seq(
-  scalaVersion := "2.13.1",
+  scalaVersion := "2.12.11",
```

Comment:
@tomrf1 and I were going to update to scala 2.13.2 because of ophan library/snyk, but I think Tom just shipped a 2.12 version for the time being.  I didn't bother with cross building here because it would complicate things, it can be added in a separate PR.

Reply:
👍